internal/store/store.go

```diff
@@ -95,7 +95,16 @@ func skipVerifyTransport() http.RoundTripper {
 	}
 }
 
-func authenticatedClient(c config.Config) (*http.Client, error) {
+type Config interface {
```

Comment:
this is the other side of the change, and defining exactly what I need here, nothing else nothing more and gives me the freedom to accept any struct that implements the interface I need

This is in line with another change coming to handle different outputs